.sassdocrc

```diff
@@ -1,11 +1,12 @@
-theme: 'herman'
+theme: herman
```

Comment:
@mirisuzanne I notice you had this pointing to your local Herman. Is there something missing in the published v3.2.0 that we need for True?

Reply:
nope, probably just from testing before we released Herman